index.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+require('env2')('.env');
+
+const Promise = require('bluebird');
+const sauce = require('mintsauce');
```

Comment:
@jruts what would be your _preferred_ pattern (_I can see both sides of the debate... want to know if there's an alternative approach we should consider_)